cmd/lookout/push.go

```diff
@@ -61,6 +62,11 @@ func (c *PushCommand) Execute(args []string) error {
 	for {
 		commit, err := log.Next()
 		if err != nil {
+			if err.Error() == "EOF" {
```

Comment:
please, when you check for error don't compare text value of it.
In this case it's `io.EOF` error. So the check should be `if err == io.EOF {`.

Reply:
:+1: done in 77dedb8.